nfts/backend/.env

```diff
@@ -8,8 +8,8 @@ USER_EMAIL_ADDRESS =
 
 // Your NFT contract address.
 // You get this after running the createCollectionAndMintToken script.
-COLLECTION = 
+COLLECTION=
```

Comment:
I would keep 3 directories
- `scripts` for one-off scripts
- `backend` for the backend server
- `frontend` for the frontend server

Reply:
And in the readme let's describe the folder structure

Reply:
Done